### ISSUE TITLE
Recommended way to query changes to a field?

### ISSUE THREAD

**Issue by ayalpani** [jagi/meteor-astronomy#269]
Hi,

in my Astronomy Collection, I have a field that I want to observe to react properly on changes. Let's say it is a boolean field named "visible", this was my naive approach:

  MyCollection = Astro.Class({
    
    // field definitions etc...

    events: {
      afterChange: function(e) {
        if (e.data.fieldName === 'visible' && e.data.operation === 'set' && e.target.visible) {
          // ... react
        }
      }
    }
  }

Basically this works fine. But is it the right way to achieve my goal? And how do I handle non-boolean fields, if I need to compare value-of-field-before to value-of-field-after?

Thanks!

**Comment by jagi** [comment]
Yes it's a right way of doing this. To get a value before or after the change, you can use `getModified()` method.
